content/yaml/building-a-native-ios-app.md

```diff
@@ -22,12 +22,28 @@ For building an unsigned iOS app (.app), you need to run the following command i
                        CODE_SIGNING_REQUIRED=NO \
                        CODE_SIGNING_ALLOWED=NO
 
+If you don't have a workspace, use `-project "MyXcodeProject.xcodeproj"` instead of the `-workspace "MyXcodeWorkspace.xcworkspace"` option.
```

Comment:
I cannot comment on non-changed lines, but the example script starts with `cd` statement to build dir, which is completely unnecessary as all commands are executed by default with `$FCI_BUILD_DIR` being the working directory.

Reply:
thank you! Fixed in 9f13727, as well as some other places where it's not redundant. I just remember I wrote something like `<you_project_directory>` but it was changed:)